sdk/storage/Azure.Storage.Files.Shares/src/Models/NfsFileMode.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Only applicable to NFS files and directories.
+    /// The mode permissions of the file or directory.
+    /// </summary>
+    public class NfsFileMode
```

Comment:
Consider adding ToSymbolicPermissions(), ToOctalPermissions(), ParseOctalPermsisions() and ParseSymbolicPermissions(), similar to Data Lake PathPermissions()